apps/aecore/lib/aecore/chain/#block_validation.ex#

```diff
@@ -0,0 +1,191 @@
+defmodule Aecore.Chain.BlockValidation do
```

Comment:
please, in no case commit commented files, git is a version control system, so delete it and recover if needed or exclude from commiting

Reply:
We are sorry, it slipped without us noticing.